src/middleware/express.ts

```diff
@@ -106,6 +106,7 @@ export async function middleware(
     ),
   };
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
nit: why not remove unused variables?

Reply:
Was wondering if this function is used anywhere else despite a fact it is not exported (to avoid any build breaks).